server/lib/models/reaction.go

```diff
@@ -10,6 +14,58 @@ type Reaction struct {
 }
 
 func InitReaction() {
-	create_table(Reaction{}, "reactions", "Id")
+	t := create_table(Reaction{}, "reactions", "Id", "Persona")
+	t.AddIndex("created_locale", "INDEX", []string{"Created", "Locale"})
 }
 
+//create fixture
+type TopicInfo struct {
+	Id proto.UUID
+	Channel proto.UUID
+	Point int64
+	Vote uint64
+}
+
+func InsertReactionFixture(dbif Dbif) error {
```

Comment:
__[gometalinter]__
* [golint] exported function InsertReactionFixture should have comment or be unexported
* [gocyclo] cyclomatic complexity 12 of function InsertReactionFixture() is high (> 10)